packages/common/src/api/comments.ts

```diff
@@ -331,6 +332,65 @@ const commentsApi = createApi({
           userId
         )
       }
+    },
+    getTrackCommentNotificationSetting: {
```

Comment:
setting a reminder for myself: I need to port this into tan-query after my PR lands